lexicons/app/bsky/graph/getFollowersYourFollow.json

```diff
@@ -0,0 +1,34 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.graph.getFollowersYouFollow",
```

Comment:
I don't have a strong opinion but I tend to lean towards specificity since "mutuals" is a kind of overloaded term that could mean like, people you both follow, people that follow both of you, and people that follow them that you also follow (like this one).